codice/alliance
# CAL-350 Update banner markings documentation

#### What does this PR do?
Backport of CAL-350 from master to Alliance 0.3.
#### Who is reviewing it? 
@vinamartin @mcalcote @ricklarsen 
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@ricklarsen 
@rzwiefel 
#### How should this be tested?
Build docs. Make sure the results include a section about the banner markings extractor.
#### Any background context you want to provide?
None.
#### What are the relevant tickets?

[CAL-350](https://codice.atlassian.net/browse/CAL-350)
#### Screenshots (if appropriate)
None
#### Checklist:
- [N/A] Update / Add Unit Tests
- [N/A ] Update / Add Integration Tests
